sdk/storage/azure-storage-queue/azure/storage/queue/_models.py

```diff
@@ -31,15 +34,15 @@ class QueueAnalyticsLogging(GeneratedLogging):
         The retention policy for the metrics.
     """
 
-    def __init__(self, **kwargs):
+    def __init__(self, **kwargs: Any) -> None:
         self.version = kwargs.get('version', '1.0')
         self.delete = kwargs.get('delete', False)
         self.read = kwargs.get('read', False)
         self.write = kwargs.get('write', False)
         self.retention_policy = kwargs.get('retention_policy') or RetentionPolicy()
 
     @classmethod
-    def _from_generated(cls, generated):
+    def _from_generated(cls, generated: Any) -> Self:
```

Comment:
Just want to call this out -- wasn't sure what to put as the return type but from my understanding `_from_generated` is just adding in the additional properties that were from the generated response, and so it is just appending to the specific object (but you can't reference its own type in its definition) so I think `Self` is sufficient here?

Reply:
LGTM, just need the funny import